File: objective-c/demo-auth-objc/DetailsViewController.m
//
//  DetailsViewController.m
//  Copyright © 2015-2016 Sequencing.com
//

#import "DetailsViewController.h"
#import "DemoDataCell.h"

@implementation DetailsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"File details";
    
    self.fileDetails.lineBreakMode = NSLineBreakByWordWrapping;
    self.fileDetails.numberOfLines = 0;
    
    self.fileDetails.text = [DemoDataCell prepareText:self.nowSelectedFile];
    NSLog(@"%@", self.fileDetails.text);
    [self.fileDetails sizeToFit];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: swift/demo-auth-swift/DetailsViewController.swift
//
//  DetailsViewController.swift
//  Copyright © 2015-2016 Sequencing.com. All rights reserved.
//

import UIKit

class DetailsViewController: UIViewController {

    @IBOutlet var fileDetails: UILabel!
    
    var nowSelectedFileType: String = ""
    var nowSelectedFile = NSDictionary()
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        self.title = "File details"
        
        self.fileDetails.lineBreakMode = NSLineBreakMode.ByWordWrapping
        self.fileDetails.numberOfLines = 0
        
        self.fileDetails.text = DemoDataCell.prepareText(self.nowSelectedFile)
        self.fileDetails.sizeToFit()
    }
    
}
